### PR TITLE
chore(debug): support configuring channels when creating or updating bot

### PR DESCRIPTION
**Pull request by kuojianlu**
```yml
  - uses: botFramework/create # Create or update the bot registration on dev.botframework.com
    with:
      botId: ${{BOT_ID}}
      name: bot
      messagingEndpoint: ${{BOT_ENDPOINT}}/api/messages
      description: ""
      channels:
        - name: msteams
        - name: outlook
```

### CODE REVIEW

File: packages/fx-core/src/component/driver/botFramework/createOrUpdateBot.ts [hunk 1]
```diff
@@ -75,13 +81,29 @@ export class CreateOrUpdateBotFrameworkBotDriver implements StepDriver {
         getLocalizedString("driver.botFramework.progressBar.createOrUpdateBot")
       );
 
+      let callingEndpoint: string | undefined = undefined;
+      let configuredChannels: BotChannelType[] | undefined = undefined;
+      if (args.channels) {
```
Comment (swatDong) [inline packages/fx-core/src/component/driver/botFramework/createOrUpdateBot.ts:86]: What if `channels` is undefined?
And what's the case to set `enable` to false for certain channel?

Comment (kuojianlu) [reply packages/fx-core/src/component/driver/botFramework/createOrUpdateBot.ts:86]: If `channels` is undefined, will not update the configured channels, see https://github.com/OfficeDev/TeamsFx/blob/kuojianlu/botFramework-channel/packages/fx-core/src/component/resource/botService/botRegistration/utils.ts#L18.
If setting `enable` to false, will disable the channel.

Comment (swatDong) [reply packages/fx-core/src/component/driver/botFramework/createOrUpdateBot.ts:86]: How about removing `enable`? To disable the channel, just to remove the whole object?

Comment (kuojianlu) [reply packages/fx-core/src/component/driver/botFramework/createOrUpdateBot.ts:86]: Good idea, `enable` seems redundant. Let me update it.

**Review by swatDong — APPROVED** [review]

File: packages/fx-core/src/component/driver/botFramework/createOrUpdateBot.ts [hunk 2]
```diff
@@ -75,13 +81,27 @@ export class CreateOrUpdateBotFrameworkBotDriver implements StepDriver {
...
+        configuredChannels = [];
+        for (const channel of args.channels) {
+          if (channel.name === BotChannelType.MicrosoftTeams) {
```
Comment (IvanJobs) [inline packages/fx-core/src/component/driver/botFramework/createOrUpdateBot.ts:89]: Just FYI: My preference would be to use `switch` here.

**Comment by IvanJobs** [comment]
Do we want to support this for V2? It looks like it only supported V3 for drivers.

**Review by IvanJobs — APPROVED** [review]

**Comment by kuojianlu** [comment]
> Do we want to support this for V2 or the remote experience? It looks like it only supported V3 for drivers.

Confirmed with Junjie, he said V2 is no need to be supported. For remote, I've already updated the bicep to configure Outlook channel, see https://github.com/OfficeDev/TeamsFx/blob/dev/templates/scenarios/js/m365-message-extension/infra/botRegistration/azurebot.bicep.

**Pull request merged**